AspNet.Security.OAuth.Providers.sln

```diff
@@ -184,6 +184,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "docs", "docs", "{C2CA4B38-A
 		docs\bitbucket.md = docs\bitbucket.md
 		docs\digitalocean.md = docs\digitalocean.md
 		docs\discord.md = docs\discord.md
+		docs\docusign.md = docs\docusign.md
+		douyin.md = douyin.md
```

Comment:
~~Should be in `docs`~~

Remove this, the document adds no settings that are different to any other OAuth provider.